modules/apps/content-dashboard/content-dashboard-web/src/main/resources/META-INF/resources/view.jsp

```diff
@@ -192,6 +193,9 @@ ContentDashboardAdminManagementToolbarDisplayContext contentDashboardAdminManage
 
 							<liferay-ui:search-container-column-text>
 								<clay:dropdown-actions
+									additionalProps='<%= HashMapBuilder.<String, Object>put(
```

Comment:
Couldn't we maybe pass this as part of the items instead of `additionalProps`?

Reply:
Mmmm, I don't think so. We need this `data-row-id` to be rendered in the dom element, which in this case is done in <clay:dropdown-actions> via otherProps
```
<ClayButton className={cssClass} {...otherProps}>
```

Reply:
Maybe... I was just looking at the `actions[action]` call and wondering why one param comes from `item.data` and the other from `otherProps`. Couldn't that be:

```javascript
actions[action]({
	fetchURL: item.data.fetchURL,
	portletNamespace,
	rowId: item.data.classPK, // or rowId
});
```

Why does the `data-row-id` need to be rendered in the button element inside the dropdown? I would expect it in the `searchContainer` row which is where you get it from with the `querySelector(...) -> tr` right?

Reply:
Maybe I'm failing to understand the HTML structure and what this is really targeting:

```javascript
	document
		.querySelector(
			`[data-searchcontainerid="${portletNamespace}content"] [data-row-id="${rowId}"]`
		)
		.closest('tr');
```

Reply:
That's the `tr` where we're adding the class `active`, but not the element where the id is being rendered. If there's a way to add data-attributes to the `searchContainer` row please let me know, I couldn't find it.

The overall idea is:
- Dropdown (and quick actions) are being renderer outside the searchContainer (through Portal I guess)
- Because of that we need to place the `classPK` as a data-attribute somewhere in the row (ideally in the `tr`, but I don't know how to do it with the search container
- Then in the propsTransformer we get that `tr` DOM element and add/remove to it the class `active`

Reply:
I think the "easiest" way is to provide a `rowData` object. Unfortunately we don't print rowid by default :(

```jsp
<liferay-ui:search-container-row
    className="com.liferay.content.dashboard.web.internal.item.ContentDashboardItem"
    keyProperty="id"
    modelVar="contentDashboardItem"
    rowIdProperty="classPK"
>

    <%
    Map<String, Object> rowData = new HashMap<String, Object>();

    rowData.put("rowId", row.getRowId());

    row.setData(rowData);
    %>

    <...>
</liferay-ui:search-container-row>
```

This should render

```
<tr data-rowid="2391">
```

You can add any data you want. It still looks weird to call this `rowId` when it's the `classPK` of the element being rendered and not defined as the `rowId` (which if you compare in the current code will be `1`, `2`...)